### DISCUSSION TITLE
is there one way to use the running app directly ?

### DISCUSSION THREAD

**Discussion by billychen2014** [appium/appium#15129] [Q&A]
hello all,

 i encountered one issue need to do like below :

1. need to launch app without appium joining firstly
2. when app home page appear , appium  find the element and do actions directly 

hmm, is there one way to implement this?

**Comment by billychen2014** [comment]
hmm, it looks like driver.activate_app ('package name'),am i right?

**Comment by KazuCocoa** [comment]
You can do like below;

1. Start appium with `appium:autoLaunch` as `false`. bundleId/appActivity is the app under test you want to handle.
2. launch the app under test in your way
3. Send command to appium which started in step 1.

**Comment by billychen2014** [comment]
got it ,  i just tested and it works, thanks a lot.
